relay-event-schema/src/protocol/attributes.rs

```diff
@@ -120,10 +120,29 @@ pub fn attribute_pii_from_conventions(state: &ProcessingState) -> Pii {
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum AttributeType {
+    /// A boolean type.
```

Comment:
I would remove the "type" from all of these.
```suggestion
    /// A boolean.
```